src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,80 @@
+<!---{
+    "experiment": true,
+    "component": "amp-experiment"
+  }--->
+<!-- 
+  #### Introduction
+
+  [amp-experiment](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md) allows to perform user experience experiments on an AMP document and collect resulting data.
```

Comment:
We should use the term A/B testing somewhere, e.g. add: "This is good for A/B testing new features in your AMPs."